django_project/certification/forms.py

```diff
@@ -149,7 +149,8 @@ def __init__(self, *args, **kwargs):
         self.helper.layout = layout
         self.helper.html5_required = False
         super(CourseConvenerForm, self).__init__(*args, **kwargs)
-        self.fields['user'].label_from_instance = lambda obj: "%s <%s>" % (obj.get_full_name(), obj)
```

Comment:
@ann26 You can set your python editor to allow only 79 characters, you will avoid these commits to fix travis ;-)